examples/source/1.components/amp-list.html

```diff
@@ -122,7 +122,8 @@
   The list content is rendered via an [amp-mustache template]({{g.doc('/content/amp-dev/documentation/components/reference/amp-mustache.md', locale=doc.locale).url.path}}). The template can be specified either by id, or by using a nested element.
```

Comment:
This is the first amp-list sample, can you mention the `binding=no` attribute here.